python/lsst/pipe/tasks/calexpCutout.py

```diff
@@ -0,0 +1,96 @@
+import lsst.pipe.base as pipeBase
+import lsst.geom as geom
+from lsst.pex.config import Field
+from lsst.meas.algorithms import Stamp, Stamps
+
+__all__ = ['CalexpCutoutTaskConfig', 'CalexpCutoutTask']
+
+
+class CalexpCutoutTaskConnections(pipeBase.PipelineTaskConnections,
+                                  dimensions=("instrument", 'visit', 'detector'),
+                                  defaultTemplates={}):
+    """Connections class for CalexpCutoutTask
+    """
+    in_cat = pipeBase.connectionTypes.Input(
+        doc="Locations for cutouts",
+        name="cutout_positions",
+        storageClass="StructuredDataDict",
+        dimensions=("instrument", 'visit', 'detector'),
+    )
+    calexp = pipeBase.connectionTypes.Input(
+        doc="Calexp objects",
+        name="calexp",
+        storageClass="ExposureF",
+        dimensions=['instrument', 'visit', 'detector'],
+    )
+    cutouts = pipeBase.connectionTypes.Output(
+        doc="Cutouts",
+        name="calexp_cutouts",
+        storageClass="Stamps",
+        dimensions=("instrument", 'visit', 'detector'),
+    )
+
+
+class CalexpCutoutTaskConfig(pipeBase.PipelineTaskConfig,
+                             pipelineConnections=CalexpCutoutTaskConnections):
+    """Configuration for CalexpCutoutTask
+    """
+    max_cutouts = Field(dtype=int, default=100, doc='Maximum number of entries to process. '
+                                                    'The result will be the first N in the catalog.')
+    skip_bad = Field(dtype=bool, default=True, doc='Skip cutouts that do not fall completely within'
+                                                   ' the calexp bounding box?')
+
+
+class CalexpCutoutTask(pipeBase.PipelineTask):
+    """Task for computing cutouts on a specific calexp given
+    positions and sizes of the stamps.
+    """
+    ConfigClass = CalexpCutoutTaskConfig
+    _DefaultName = "calexpCutoutTask"
+
+    def run(self, in_cat, calexp):
+        """Compute and return the cutouts.
+
+        Parameters
+        ----------
+        in_cat : `dict`
+            A dictionary containing at least the following columns: ra, dec, size.
+            The coordinates should be in ICRS degrees.  The size is in pixels.
```

Comment:
I think "colums" is fine for a structured array, though I don't actually know the correct idiom. Since you ended up with a table, columns seems fully appropriate.